gcp/scripts/check_server_online.sh

```diff
@@ -1,13 +1,4 @@
 #!/bin/bash
 
-kibana_online=false
-while [ $kibana_online == false ]; do
-  content=$(curl -s http://elastic:changeme@localhost:5601/api/status)
-  state=$( jq -r  '.status.overall.state' <<< "${content}" )
-  if [[ $state == 'green' ]]
-  then
-    kibana_online=true
-  else
-    sleep 5
-  fi;
-done
\ No newline at end of file
+kibana_syslog=/var/log/syslog
+grep -m 1 "Kibana is now available" <(tail -f -n +1 $kibana_syslog)
```

Comment:
Due to an API change this did no longer work, so I decided to no longer depend on the API